cs.json

```diff
@@ -0,0 +1,348 @@
+            "searching" : "Hledání",
+            "controls" : {
+                "heading" : "Filtr:",
+                "placeholder" : "Hledat položku...",
+                "type" : "Typ položky",
+                "rarity" : "Vzácnost",
+                "season" : "Sezóna",
+                "apply" : "Použít &rarr;",
+                "sort" : {
+                    "heading" : "Seřadit podle",
+                    "release" : "Datum vydání",
+                    "occurrences" : "Počtu výskytů",
+                    "seen" : "Naposledy viděno",
+                    "trending" : "Zájmu",
+                    "asc-word" : "Abecedně vzestupně (od A do Z)",
+                    "desc-word" : "Abecedně sestupně (Od Z do A)",
+                    "asc-number" : "Nejvyšší",
+                    "desc-number" : "Nejnižší",
+                    "asc-date" : "Nejnovější",
+                    "desc-date" : "Nejstarší",
+                    "asc-seen" : "Naposledy viděno",
+                    "desc-seen" : "Nejdelší čekání"
+                }
+            },
+            "results" : {
+                "type" : "Typ",
+                "rarity" : "Vzácnost",
+                "season" : "Sezóna",
+                "all" : "Zobrazuji všech [number] Fortnite [noun]",
+                "query" : "Zobrazuji [result] odpovídající [query]",
+                "filter-start" : "Zobrazuji [result]",
+                "filter-end" : "Filtruji podle [filters]",
+                "no-results" : "Těmto filtrům neodpovídají žádné výsledky.",
+                "error-down-title" : "Vyhledávač kosmetiky není momentálně dostupný.",
+                "error-down-text" : "Nepodařilo se připojit k vyhledávacím serverům",
+                "error-misc-title" : "Vyhledávání nebylo možné dokončit",
+                "error-timeout" : "Požadavek vypršel",
+                "error-ratelimit" : "Příliš mnoho požadavků, zkuste to prosím později"
+            }
+        },
+        "shop" : {
+            "title" : "Aktuální rotace",
+            "title-featured" : "Doporučené položky",
+            "title-daily" : "Denní položky",
+            "error" : "Obchod není dostupná, zkuste to prosím později",
+            "line1" : "Aktuální rotace položek pro Fortnite: Battle Royale - se aktualizuje denně v [00:00 UTC](https://time.is/UTC|tooltip=Aktuální čas v UTC|newtab=true).",
+            "line2" : "Včerejší rotaci položek si můžete prohlédnout [zde](/shop/yesterday|tooltip=Včerejší rotace položek).",
+            "line3" : "Klikněte na kosmetickou položku pro zobrazení více informací o položce. Podělte se o svůj názor na aktuální rotaci hlasováním v dolní části této stránky.",
+            "countdown" : {
+                "title" : "Nové položky:",
+                "wordyFormat" : "[number] [noun]",
+                "hour-singular" : "hodina",
+                "hours-plural" : "hodin" //can be hodin or hodiny (1 hodina, 2-4 hodiny, 5-99 hodin)
```

Comment:
Unfortunately, we can't cover this special case with the way translations work at the moment. So let's go with "hodin".